equipment-service/src/main/java/com/telecominfraproject/wlan/equipment/controller/EquipmentController.java

```diff
@@ -275,19 +275,37 @@ private void validateChannelNum(Equipment equipment) {
 					Integer backupChannelNum = elementRadioConfig.getBackupChannelNumber();
 					Integer manualBackupChannelNum = elementRadioConfig.getManualBackupChannelNumber();
 					
-					List<Integer> allowedChannels = elementRadioConfig.getAllowedChannelsPowerLevels().stream().map(ChannelPowerLevel::getChannelNumber).collect(Collectors.toList());
+					List<Integer> allowedChannels = elementRadioConfig.getAllowedChannelsPowerLevels().stream().
+							map(ChannelPowerLevel::getChannelNumber).collect(Collectors.toList());
 					
 					if (allowedChannels != null && !allowedChannels.isEmpty()) {
 						checkAllowedChannels(channelNum, "channelNumber", allowedChannels);
+						backupChannelNum = correctOldInvalidBackupChannel(radioType, backupChannelNum, elementRadioConfig, false);
 						checkAllowedChannels(backupChannelNum, "backupChannelNumber", allowedChannels);
+						
 						checkAllowedChannels(manualChannelNum, "manualChannelNumber", allowedChannels);
+						manualBackupChannelNum = correctOldInvalidBackupChannel(radioType, manualBackupChannelNum, elementRadioConfig, true);
 						checkAllowedChannels(manualBackupChannelNum, "manualBackupChannelNumber", allowedChannels);
 					}
 				}
 			}
 		}
 	}
 	
+	// Correct old out of range default backup channel number setting (154) on 5GU
+	private Integer correctOldInvalidBackupChannel(RadioType radioType, Integer channelNum,
+			ElementRadioConfiguration elementRadioConfig, boolean manual) {
+		if (radioType == RadioType.is5GHzU && channelNum != null && channelNum == 154) {
```

Comment:
So, whoever configures the channel 154 would actually receive configuration for 157 ?
Is this correct behaviour for all country codes? or only for us/canada ?
Is this permanent change or a transitionary? 
If it is transitionary - we should immediately schedule a JIRA for the future to remove the temporary code.
If that is a permanent solution, we should at least include a big warning in the log whenever this happens with explanations why it is happening.